test/sanity/ignore.txt

```diff
@@ -225,3 +225,9 @@ test/units/module_utils/facts/test_facts.py mypy-3.8:assignment
 test/units/modules/mount_facts_data.py mypy-3.8:arg-type
 test/units/modules/test_apt.py mypy-3.8:name-match
 test/units/modules/test_mount_facts.py mypy-3.8:index
+test/units/module_utils/test_deps.py mypy-3.8:import-not-found
+test/units/module_utils/test_deps.py mypy-3.9:import-not-found
+test/units/module_utils/test_deps.py mypy-3.10:import-not-found
+test/units/module_utils/test_deps.py mypy-3.11:import-not-found
+test/units/module_utils/test_deps.py mypy-3.12:import-not-found
+test/units/module_utils/test_deps.py mypy-3.13:import-not-found
```

Comment:
is there a way to achieve this w/o having to add ignores for mypy? This gets cumbersome for collections as they require a ignore per ansible version supported.

Reply:
hi @bcoca thanks reviewing, I will look into that (I tried briefly while submitting the PR, but eventually took the ~~lazy~~ easy way out).

That being said, not sure I understand why collections would need to do this for unit test files from core. In community.general at least I do believe we do not include core's unit test files in the checks. Why would any collection do that in the first place?

Reply:
I must add that there are **many** lines in the ignore file for mypy, so that logic should potentially apply to many other parts of the code.